scheduler/src/main/java/org/cloudfoundry/autoscaler/scheduler/service/ScalingScheduleManager.java

```diff
@@ -0,0 +1,162 @@
+package org.cloudfoundry.autoscaler.scheduler.service;
+
+import java.sql.Time;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.cloudfoundry.autoscaler.scheduler.dao.ScheduleDao;
+import org.cloudfoundry.autoscaler.scheduler.entity.ScheduleEntity;
+import org.cloudfoundry.autoscaler.scheduler.rest.model.ApplicationScalingSchedules;
+import org.cloudfoundry.autoscaler.scheduler.rest.model.SpecificDateSchedule;
+import org.cloudfoundry.autoscaler.scheduler.util.DateHelper;
+import org.cloudfoundry.autoscaler.scheduler.util.JobScheduleTypeEnum;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+/**
+ * Service class to persist the schedule entity in the database and create
+ * scheduled job.
+ * 
+ * @author Fujitsu
+ *
+ */
+@Service
+public class ScalingScheduleManager {
+
+	@Autowired
+	private ScheduleDao scheduleDao;
+	@Autowired
+	private ScalingJobManager scalingJobManager;
+	private Log logger = LogFactory.getLog(this.getClass());
+
+	/**
+	 * TODO: Created just a place holder, method will require to call dao and
+	 * fetch all the schedules for the specified application id.
+	 * 
+	 * @param appId
+	 * @return
+	 * @throws Exception
+	 */
+	public ApplicationScalingSchedules getSchedules(String appId) {
+		List<ScheduleEntity> allScheduleEntitiesForApp = scheduleDao.findAllSchedulesByAppId(appId);
+		ApplicationScalingSchedules applicationScalinSchedules = populateScheduleModel(appId,
+				allScheduleEntitiesForApp);
+		return applicationScalinSchedules;
+	}
+
+	/**
+	 * Calls private helper methods to persist the schedules in the database and
+	 * calls ScalingJobManager to create scaling action jobs.
+	 * 
+	 * @param applicationScalinSchedules
+	 * @throws Exception
+	 */
+	@Transactional
+	public String createSchedules(ApplicationScalingSchedules applicationScalinSchedules) {
+
+		String appId = applicationScalinSchedules.getApp_id();
+		String timeZone = applicationScalinSchedules.getTimezone();
+		List<SpecificDateSchedule> specificDateSchedules = applicationScalinSchedules.getSpecific_date();
+		try {
+			for (SpecificDateSchedule specificDateSchedule : specificDateSchedules) {
+
+				// Set up the schedule entity from model object
+				ScheduleEntity scheduleEntity = populateScheduleEntity(appId, timeZone, specificDateSchedule,
+						JobScheduleTypeEnum.SIMPLE);
+
+				// Persist the schedule in database
+				saveNewSpecificDateSchedule(scheduleEntity);
+
+				// Ask ScalingJobManager to create scaling job
+				scalingJobManager.createSimpleJob(scheduleEntity);
+			}
+		} catch (Exception x) {
+			logger.error(x.getMessage(), x);
```

Comment:
The exception message will be logged with x, pass in something more concrete like create-schedules failed or something.